starknet_py/common.py

```diff
@@ -97,7 +97,15 @@ def create_casm_class(compiled_contract: str) -> CasmClass:
     :param compiled_contract: contract compiled using starknet-sierra-compile.
     :return: CasmClass instance.
     """
-    return cast(CasmClass, CasmClassSchema().loads(compiled_contract))
+    try:
+        return cast(CasmClass, CasmClassSchema().loads(compiled_contract))
+    except ValidationError as err:
+        if err.messages == {"pythonic_hints": ["Missing data for required field."]}:
```

Comment:
How about adding `Error: Libfunc alloc_local is not allowed in the libfuncs list` too?

Reply:
I think it's not necessary, as this is included in the error message:
```
Run with '--allowed-libfuncs-list-name experimental_v0.1.0' to allow all libfuncs.
```